linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_configure_sufficiently_large_partition/oval/shared.xml

```diff
@@ -0,0 +1,34 @@
+{{% if target_oval_version >= [5, 11.2] %}}
```

Comment:
> This is because of `block_size` right?
> Could you add a comment about this?

Yes. Looks like 'block_size' is only available in 5.11.2. https://github.com/OVALProject/Language/blob/5.11.2/schemas/linux-definitions-schema.xsd

Reply:
> @guangyee Are you specifying `SSG_TARGET_OVAL_MINOR_VERSION` as `11.2` during build??

No. The rule is currently evaluated as `nochecked` because the openSCAP version we have on our SLES12 test box supports version 5.11.1. `OVAL Version: 5.11.1`. This rule was originally copied over from our internal development branch, which was validated by my predecessor. We want to enable it at least users can use the rule description/fix text to manually remediate.

@abergmann do you have any historic context on this one?

Reply:
The only way to get `OVAL Version: 5.11.2` is by using `-DSSG_TARGET_OVAL_MINOR_VERSION=11.2` during build.

Without the `block_size` value we cannot calculate the correct disk size and therefore the OVAL check is pointless when running OVAL version prior to 5.11.2.

Reply:
Thank you for the clarifications.